resources/js/Pages/Results.vue

```diff
@@ -1,28 +1,51 @@
 <template>
     <div class="page-container results-page">
         <Head title="Mismatch Finder - Results" />
-        <section id="results" v-if="Object.keys(results).length">
-            <section class="item-mismatches" v-for="(mismatches, item, idx) in results" :key="idx">
+        <div v-if="Object.keys(results).length < item_ids.length" class="not-found">
```

Comment:
Nice way to determine if the message should show!

Reply:
:D